pandas/core/generic.py

```diff
@@ -113,10 +114,10 @@ class NDFrame(PandasObject, SelectionMixin):
                        '_default_fill_value', '_metadata', '__array_struct__',
                        '__array_interface__']
     _internal_names_set = set(_internal_names)
```

Comment:
can you type the rest of these?

Reply:
@jreback Sure, I will get it done tomorrow morning(gotta sleep now :sleeping: ).

Reply:
@jreback done.